airflow/www/app.py

```diff
@@ -70,7 +72,27 @@
 
 def create_app(config=None, testing=False):
     """Create a new instance of Airflow WWW app."""
-    flask_app = Flask(__name__)
+    connexion_app = connexion.FlaskApp(__name__)
+
+    @connexion_app.app.before_request
+    def before_request():
+        """Exempts the view function associated with '/api/v1' requests from CSRF protection."""
+        if request.path.startswith("/api/v1"):  # TODO: make sure this path is correct
```

Comment:
Answering the question about the paths here - this execamption shoult be done for all the API endpoints - so basicaly anything that is stored in ` init_views.base_paths` -> all the API  endpoints will register itself there - for example:

```python
    base_path = "/api/v1"
    base_paths.append(base_path)
```

And and - we should be able to add exemption when path starts with any of the api urls registered there.

Reply:
So this is OK to exempt like this
```python3
request.path.startswith("/api/v1")
```